samples/WebAPI/Program.cs

```diff
@@ -3,11 +3,19 @@
 
 using System.Text.Json.Serialization;
 using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Worker;
 
-var builder = WebApplication.CreateBuilder(args);
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 
 // Add all the generated tasks
-builder.Services.AddDurableTask(taskRegistry => taskRegistry.AddAllGeneratedTasks());
+builder.Services.AddDurableTaskWorker(builder =>
+{
+    builder.AddTasks(r => r.AddAllGeneratedTasks());
+    builder.UseGrpc();
+});
+
+builder.Services.AddDurableTaskClient(b => b.UseGrpc().RegisterDirectly());
```

Comment:
Is the `.RegisterDirectly()` call still required after some of your more recent changes?

Reply:
It is not, removed.